contracts/Tokenomics.sol

```diff
@@ -1373,14 +1371,17 @@ contract Tokenomics is TokenomicsConstants {
         // Recalculate maxBond, which is issued as a credit for the upcoming epoch and is part of the effectiveBond
         uint256 curMaxBond = (curEpochLen * curInflationPerSecond * tp.epochPoint.maxBondFraction) / 100;
 
+        emit InflationPerSecondFractionsUpdated(curInflationPerSecond, effectiveBond,
+            mapEpochStakingPoints[curEpochCounter].stakingIncentive, maxBondFraction, topUpComponentFraction,
+            topUpAgentFraction, stakingFraction);
```

Comment:
Adding effective bond and staking incentive reset values into the event.

Reply:
why change in order of storage update and event?

Reply:
We want to record how much of bond and staking values have been reset.